### ISSUE TITLE
you do not have the required level to see

### ISSUE THREAD

**Issue by laemidren** [asciimoo/searx#345]
Why is safe search activated by default for image search?
I dislike when someone choose for me what I can see or not!
if you have children, you need to stay close if you want censoring, you should not trust anything than you, to choose what is good for him.

I don't want manage some prefs with cookies or another, because I flush all cookies when I close my web browser.

it is not easy to browse Web without any javascript activated, I accept only native javascript (external javascript is disgusting, beurk!)

I appreciate searx engine because is good to have answers from many search engine.

sorry about my approximative english :p

**Comment by asciimoo** [comment]
I agree, safe search should be turned off by default (or at least a config option to set default value per instance). It's another filter.. Nice to have optionally, but it's the first thing what i turn off
